src/set_theory/game/basic.lean

```diff
@@ -550,10 +539,14 @@ theorem zero_lf_inv' : ∀ (x : pgame), 0 ⧏ inv' x
 def inv'_zero : inv' 0 ≡r 1 :=
 begin
   change mk _ _ _ _ ≡r 1,
-  refine ⟨_, _, λ i, _, is_empty_elim⟩; dsimp,
-  { apply equiv.equiv_punit },
-  { apply equiv.equiv_pempty },
-  { simp }
+  refine ⟨_, _, λ i, _, is_empty.elim _⟩,
```

Comment:
There's a bit of a regression here, as for whatever reason Lean is now unable to see through some quite dense def-eqs. But given how particularly messy this proof is, I wouldn't worry much.